src/main/kotlin/com/terraformation/backend/customer/api/AdminController.kt

```diff
@@ -829,6 +830,7 @@ class AdminController(
       @RequestParam model: String,
       @RequestParam name: String?,
       @RequestParam protocol: String?,
+      @RequestParam parentId: DeviceId?,
```

Comment:
Definitely a bit of a nitpick, but move this up a line so the request parameters stay in alphabetical order. Same with the new `DevicesRow` constructor argument below this.